plugin/simple-todo.vim

```diff
@@ -36,6 +40,11 @@ endfu " }}}
 nnore <silent> <Plug>(simple-todo-new) i[ ]<space>
 inore <silent> <Plug>(simple-todo-new) [ ]<space>
 
+" Create a new item with some list prefix symbol
+nnore <silent> <Plug>(simple-todo-new-list-item) "=g:simple_todo_list_symbol.' [ ] '<C-M>pa
+inore <silent> <Plug>(simple-todo-new-list-item) <Esc>"=g:simple_todo_list_symbol.' [ ] '<C-M>pa
+
```

Comment:
- Please use `<cr>` instead of `<C-M>` (they both are equal, keep `<cr>` for consistency with other lines);
- Remove extra blank line (for consistency as well).